dashboards/victoriametrics.json

```diff
@@ -1838,18 +1837,25 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(go_gc_duration_seconds_sum{job=\"$job\"}[2m]))",
+          "expr": "irate(process_cpu_seconds_total{job=\"$job\"}[1m])",
```

Comment:
Leave only `rate(process_cpu_seconds_total{job="$job"}[1m])` graph. Remove `irate` graph because of https://medium.com/@valyala/why-irate-from-prometheus-doesnt-capture-spikes-45f9896d7832

Reply:
Tried to make it similar to Prometheus dashboard. 
Removed `irate`